types/src/main/scala/aqua/types/CompareTypes.scala

```diff
@@ -63,19 +63,21 @@ object CompareTypes {
     val lfView = lf.view
     val rfView = rf.view
     if (lf == rf) 0.0
-    else if (
-      lf.keys.forall(rf.contains) && compareTypesList(
-        lfView.values.toList,
-        rfView.filterKeys(lfNEM.keys.contains).values.toList
-      ) == 1.0
-    ) 1.0
-    else if (
-      rf.keys.forall(lf.contains) && compareTypesList(
-        lfView.filterKeys(rfNEM.keys.contains).values.toList,
-        rfView.values.toList
-      ) == -1.0
-    ) -1.0
-    else NaN
+    else if (lf.keys.forall(rf.contains)) {
+      if (
+        compareTypesList(
+          lfView.values.toList,
+          rfView.filterKeys(lfNEM.keys.contains).values.toList
+        ) >= 0.0
+      ) 1.0
+      else if (
+        compareTypesList(
```

Comment:
Is this call correct?
We know that `rf` contains all `lf` fields, but not the opposite